ui/k-info-panel/show_circuit.kytos

```diff
@@ -62,9 +62,12 @@
                 <th>{{cindex}}</th>
                 <td>
                   <k-input v-if="column.editable" :id="column.name" :value.sync="column.value"></k-input>
-                  <span v-else>
+                  <span v-else-if="column.editable === false">
                     {{column.value}}
                   </span>
+                  <k-dropdown v-else-if="column.name === 'queue_id'" :options="queue_options(column)"
```

Comment:
Great enhancement. Awesome to have this `k-dropdown`.

Reply:
Thanks